location_history_json_converter.py

```diff
@@ -56,6 +73,11 @@ def main(argv):
             return
 
         items = data["locations"]
+        
+        if args.chronological:
+            if args.startdate or args.enddate:
+                items = [ item for item in items if dateCheck(item["timestampMs"]) ]
```

Comment:
I think the filtering for `startdate`, `enddate` should be moved out of the `args.chronological` branch, which would have the advantage that all the `if not dateCheck(item["timestampMs"])  :  continue` lines can be removed